### PR TITLE
remove no-schema config

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove all the `no-schema` config of Lightning  due to it can  automatically detect now.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ ADMIN CHECKSUM TABLE `schema`.`table`;
 
 ## 我已经在下游创建好库和表了，TiDB Lightning 可以忽略建库建表操作吗？
 
-可以。在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
+自 v5.4 版本之后可以自动识别。低于 v5.4 版本需在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-faq.md:80]: 这个行为可能 v5.0 就带入了

**Review by gozssky — COMMENTED** [review]
Rest LGTM.

File: tidb-lightning/tidb-lightning-faq.md (hunk 1 above)
Comment (gozssky) [reply tidb-lightning/tidb-lightning-faq.md:80]: 确认了一下， v5.1 就改成这个行为了

Comment (ran-huang) [reply tidb-lightning/tidb-lightning-faq.md:80]: @sunzhaoyang

Comment (sunzhaoyang) [inline tidb-lightning/tidb-lightning-faq.md:80]:
```suggestion
自 v5.1 版本之后可以自动识别。低于 v5.1 版本需在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
```

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ ADMIN CHECKSUM TABLE `schema`.`table`;
 
 ## 我已经在下游创建好库和表了，TiDB Lightning 可以忽略建库建表操作吗？
 
-可以。在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
+自 v5.1 版本之后可以自动识别。低于 v5.1 版本需在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-faq.md:80]:
```suggestion
自 v5.1 起，TiDB Lightning 可以自动识别下游的库和表。如果你使用低于 v5.1 的 TiDB Lightning，需在配置文档中的 `[mydumper]` 部分将 `no-schema` 设置为 `true` 即可。`no-schema=true` 会默认下游已经创建好所需的数据库和表，如果没有创建，会报错。
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -297,7 +297,7 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
 **解决办法**：
 
 1. 编辑数据源，保存为纯 UTF-8 或 GB-18030 的文件。
-2. 手动在目标数量库创建所有的表，然后设置 `[mydumper] no-schema = true` 跳过创建表的步骤。
+2. 手动在目标数量库创建所有的表。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-faq.md:300]:
```suggestion
2. 手动在目标数据库创建所有的表。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**